.env.local.example

```diff
@@ -25,6 +25,6 @@ FEATURE_CAMPAIGN=true
 ## Keycloak ##
 ##############
 
-KEYCLOAK_URL=https://keycloak.podkrepi.bg/auth/
+KEYCLOAK_URL=http://localhost:8080/auth
```

Comment:
I felt like that should be like changed, but should we leave it as `http://localhost:8080/auth` or should it be like `YOUR_KEYLOAK_AUTH_URL`

Reply:
We better have a real URL for the local keycloak instead of fake string

Many people have `.env` symlinked